intranet/templates/eighth/admin/transfer_students.html

```diff
@@ -38,9 +48,14 @@
         {% else %}
             <p>Do you want to transfer <b>{{ num }} students</b> from:</p>
             <p><b>{{ source_act }}</b></p>
-            to:<br><br>
+            to:<br>
```

Comment:
Why was this `<br>` removed?

Reply:
The space was weird. You can see it without a dev env at https://ion.tjhsst.edu/eighth/admin/scheduling/transfer_students_action?source_act=135085&dest_act=133549. This just moves that bottom line up.